F/FLEXPART/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FLEXPART"
+version = v"10.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://www.flexpart.eu/gitmob/flexpart", "3d7eebf7c4909f59db5ec32c524f88fb846e9fe5"),
+    GitSource("https://www.flexpart.eu/gitmob/flex_extract", "e0005c99ac81d12faa45a8ff799debbd592b0dc0"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+# The flexpart-parmod.patch modify some parameters to extend the size of 
+# the meteorological input files and the maximum number of particules that FLEXPART can handle.
+script = raw"""
+cd flexpart
+atomic_patch -p1 /workspace/srcdir/patches/flexpart-makefile.patch
+atomic_patch -p1 /workspace/srcdir/patches/flexpart-parmod.patch
+cd src
+make ncf=yes
+cp FLEXPART $bindir
+
+cd ../../flex_extract
+atomic_patch -p1 /workspace/srcdir/patches/flexextract-makefile.patch
+cd Source/Fortran/
+make -f makefile_local_gfortran
+cp calc_etadot_fast.out $bindir/calc_etadot
+"""
+
+platforms = supported_platforms()
```

Comment:
NetCDFF is available only for a handful of platforms: https://github.com/JuliaPackaging/Yggdrasil/blob/8c3c38cf640244b7ef05518dfe2b2a1b3406ad23/N/NetCDFF/build_tarballs.jl#L29-L35 because of limitations in the HDF5 distribution.  Also, there are plenty of other problems with this package, for example it doesn't look it supports anything but x86_64?

Reply:
Hi @giordano and thanks for your help! Yes I had problems with other platforms locally as well. Would you be ok to stay only with `Platform("x86_64", "linux"; libc = "glibc")` for the moment, and I'll see later on what can be done for the other platforms? (I'm not a developer of this package, I'm just a user trying to interface it in Julia)

Reply:
Sorry, I had missed this message

> Would you be ok to stay only with `Platform("x86_64", "linux"; libc = "glibc")` for the moment, and I'll see later on what can be done for the other platforms? (I'm not a developer of this package, I'm just a user trying to interface it in Julia)

Yes, that's fine, the decision of the platforms to support is up to you

Reply:
No problem, I commited the restriction to Platform("x86_64", "linux"; libc = "glibc").
thanks!